src/components/newTodoForm/newTodoForm.tsx

```diff
@@ -0,0 +1,128 @@
+/* eslint-disable @typescript-eslint/no-shadow */
+/* eslint-disable react-hooks/rules-of-hooks */
+import React, { ChangeEvent, FormEvent, useState } from 'react';
+import { Todo, User } from '../../types';
+import classNames from 'classnames';
+// import { todo } from 'node:test';
+
+interface NewTodoFormProps {
+  users: User[];
+  onAdd: (todo: Omit<Todo, 'id'>) => void;
+}
+
+interface Errors {
+  title?: string;
+  userId?: string;
+}
+
+// const getMaxTodoId= (todos: Todo[]) => {
+//   const ids = todos.map(todo => todo.id);
+
+//   return Math.max(...ids, 0);
+// };
```

Comment:
The `getMaxTodoId` function is commented out. If it's not being used, consider removing it to keep the code clean.